src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (value) => {
+    if (value !== undefined) {
+      sum += value;
+
+      return adder;
+    } else {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+  };
```

Comment:
`return `will break the execution anyway, so there is no need to use `else `
```suggestion
    if (value !== undefined) {
      sum += value;

      return adder;
    } 
    
    const result = sum;
    sum = 0;

    return result;
  };
```

Reply:
done